src/components/MaPagination/MaPagination.vue

```diff
@@ -7,7 +7,7 @@
         class="ma-pagination__button ma-pagination__button--backwards"
         @click="onButtonClick(currentPage - 1)"
       >
-        <ma-icon icon="Arrow" width="16" height="16" />
+        <ma-icon icon="ChevronLeft" size="medium" title="Back" />
```

Comment:
I don't think we need a title here, as the element is nested inside a Button which already has an ARIA label so it will be properly announced:

```suggestion
        <ma-icon icon="ChevronLeft" size="medium" />
```

if we still want to use a title (If I'm missing something), then I'd use the same label:

```suggestion
        <ma-icon icon="ChevronLeft" size="medium" :title="leftButtonAria" />
```

again, I believe this is not needed.

_(same thing with the button on the right)_